src/http/restApiRandomQuote.ts

```diff
@@ -3,10 +3,10 @@ import axios from 'axios';
 import { REST_API__RANDOM_QUOTE } from 'src/http/restApiURL';
 import Quote, { ServerQuoteData } from 'src/models/Quote';
 
-const restApiGetRandomQuoteByCharacterName = async (characterName: string): Promise<Quote> => {
+const restApiGetRandomQuoteByCharacterName = async (characterName: string): Promise<Quote | null> => {
   const enCodedCharacterName = characterName.replace(' ', '+');
-  const response = await axios.get(`${REST_API__RANDOM_QUOTE}?autor=${enCodedCharacterName}`);
-  return new Quote(response.data[0] as ServerQuoteData);
+  const response = await axios.get(`${REST_API__RANDOM_QUOTE}?author=${enCodedCharacterName}`);
+  return response.data.length > 0 ? new Quote(response.data[0] as ServerQuoteData) : null;
```

Comment:
If the character has quotes, the response's body is an array of quotes. If not, the response is an empty array.